src/app/vault/add-edit.component.html

```diff
@@ -308,7 +308,7 @@ <h2 class="modal-title">{{title}}</h2>
                 </div>
                 <h3 class="mt-4">{{'customFields' | i18n}}</h3>
                 <ng-container *ngIf="cipher.hasFields">
-                    <div class="row" appBoxRow *ngFor="let f of cipher.fields; let i = index">
+                    <div class="row" appBoxRow *ngFor="let f of cipher.fields; let i = index; trackBy:trackByFunction">
```

Comment:
You're dealing with different types here. So you should probably change the function in jslib to `any` type.

Reply:
You're right, of course. It was only later on that I remembered that custom fields were an issue, too :roll_eyes:

Reply:
No worries. I fixed it :)